### DISCUSSION TITLE
Support for Custom Trail Colors

### DISCUSSION THREAD

**Discussion by Pitatochu** [ToniMacaroni/SaberFactory#120] [Ideas]
Either a way to change them in the mod, or just let it take it from the saber itself (see image)
![Screenshot 2022-01-15 182721](https://user-images.githubusercontent.com/73562630/149642465-51321d66-7d56-43c2-80e4-f96f41f9cabd.png)
.

**Comment by ToniMacaroni** [comment]
The idea was to configure the entire look (including colors) and have them customizable in the mod through the shader itself.
That way there aren't multiple settings that do the same thing and creators have the choice to completely "program" their customizability.
